backends/cuda/ceed-cuda-qfunction-load.cpp

```diff
@@ -77,6 +77,7 @@ extern "C" int CeedCudaBuildQFunction(CeedQFunction qf) {
 
   // Defintions
   code << "\n#define CEED_QFUNCTION(name) inline __device__ int name\n";
+  code << "#define CEED_QFUNCTION_HELPER(name) inline __device__ int name\n";
```

Comment:
Shall we just make it
```c
#define CEED_QFUNCTION_HELPER inline __device__
```
so "helpers" can return something other than `int`?

Reply:
It is my understanding that HIP sometimes ignores `__inline__` unless we force it. I was 50-50 on making the helper return an error code or making backends force inlining. I suppose it is easier to specify behavior on backend devs.